api/backend/ecs/load_balancer_manager.go

```diff
@@ -304,17 +303,23 @@ func (e *ECSLoadBalancerManager) createLoadBalancer(
 		return err
 	}
 
-	securityGroup, err := e.upsertSecurityGroup(ecsLoadBalancerID, ports)
-	if err != nil {
-		return err
+	// only public load balancers get an additional security group
```

Comment:
Not that I've seen so far. I ran the system tests to verify:
- Public->Private communication without environment linking fails
- Public->Private communication with linking succeeds 

I also spun up a consul cluster behind a private lb and it was able to communicate to itself as expected.